views/js/qtiCreator/widgets/item/Widget.js

```diff
@@ -66,7 +66,9 @@ define([
         if(!config || !config.uri){
             throw new Error('missing required config parameter uri in item widget initialization');
         }
-
+        
+        globalConfig = config;
```

Comment:
This creates potential memory leaks.
Here you have access to an object context, so you can bind the config or the url, directly to the context.